routes/web.php

```diff
@@ -49,6 +49,11 @@
     });
 });
 
+Route::group(['prefix' => 'beatmappacks'], function () {
```

Comment:
I had it using querystrings before, but I'm not sure if we're going to continue using querystrings 🤷‍♀️ 🤷‍♂️